internal/models/integration.go

```diff
@@ -80,9 +94,59 @@ func ConvertToIntegrationModel(ctx context.Context, integration *apono.Integrati
 		}
 	}
 
+	data.ResourceOwnerMappings = ConvertResourceOwnersMappingToModel(integration.ResourceOwnersMappings)
+	data.IntegrationOwners = ConvertIntegrationOwnerToData(integration.IntegrationOwners)
+
 	return &data, nil
 }
 
+func ConvertIntegrationOwnerToData(Owners []aponoapi.IntegrationOwnerTerraform) []IntegrationOwner {
+	if Owners == nil {
+		return nil
+	}
+	var integrationOwners = make([]IntegrationOwner, len(Owners))
+	for i, in := range Owners {
+		var AttributeValue = make([]types.String, len(in.AttributeValue))
```

Comment:
```suggestion
		var attributeValue = make([]types.String, len(in.AttributeValue))
```
camel case for variable names, please adjust the rest of the code accordingly.